### ISSUE TITLE
It would really help if stacked build requests in Jenkins were automatically cancelled

### ISSUE THREAD

**Issue by alvacouch** [hydroshare/hydroshare#1965]
@mjstealey Part of the delay in Jenkins now is that if I issue two "push" requests for the same PR, the test for the first must complete before the test for the second begins. I thus end up waiting for myself. Ideally, if I issue multiple "push" requests for the exact same branch, only the last one should run. At the least, intermediate tests should not even be run.

**Comment by cuahsimarko** [comment]
@alvacouch Multiple Build Executors have been added by Phu to allow for PRs to test in parallel (four). The New Jenkins https://new-hs-ci.hydroshare.org has many admins (any staff with a RENCI login) so developers can be courteous and cancel multiple builds (if they've pushed several times). There is still the underlying issue of 3hr unit testing, which implicates iRODS, but only by attribution. The ticket for slow unit tests is here: https://github.com/hydroshare/hydroshare/issues/2421

Once we cut over the "pull request" Jenkins job to New Jenkins, may I close this GH Issue, #1965?

**Comment by cuahsimarko** [comment]
@alvacouch I'm closing this in favor of the more industry standard approach, going forward.
